source/braille.py

```diff
@@ -1760,9 +1760,10 @@ def __init__(self, handler):
 		#: The translated braille representation of the entire buffer.
 		#: @type: [int, ...]
 		self.brailleCells = []
-		#: The position in L{brailleCells} where the display window starts (inclusive).
-		#: @type: int
-		self.windowStartPos = 0
+		#: A list representing the rows in the braille window,
+		#: each item being a tuple of start and end braille buffer offsets.
+		#: Splitting the window into independent rows allows for optional avoidance of  splitting words across rows.
+		self._windowRowBufferOffsets: list[tuple[int, int]] = [(0, 0)]
```

Comment:
Shouldn't this be a variable doc string nowadays?
```suggestion
		self._windowRowBufferOffsets: list[tuple[int, int]] = [(0, 0)]
		"""A list representing the rows in the braille window,
		each item being a tuple of start and end braille buffer offsets.
		Splitting the window into independent rows allows for optional avoidance of  splitting words across rows.
		"""
```